docs/en/query_language/system.md

```diff
@@ -4,31 +4,36 @@
 - [FLUSH DISTRIBUTED](#query_language-system-flush-distributed)
 - [START DISTRIBUTED SENDS](#query_language-system-start-distributed-sends)
 
+
 ## Managing Distributed Tables {#query_language-system-distributed}
 
-ClickHouse can manage [distributed](../operations/table_engines/distributed.md) tables. When a user inserts data into such table, ClickHouse creates a queue of the data which should be sent to servers of the cluster, then asynchronously sends them. You can control the processing of queue by using the requests [STOP DISTRIBUTED SENDS](#query_language-system-stop-distributed-sends), [FLUSH DISTRIBUTED](#query_language-system-flush-distributed) and [START DISTRIBUTED SENDS](#query_language-system-start-distributed-sends). Also, you can synchronously insert distributed data with the `insert_distributed_sync` setting.
+ClickHouse can manage [distributed](../operations/table_engines/distributed.md) tables. When a user inserts data into these tables, ClickHouse first creates a queue of the data that should be sent to cluster servers, then asynchronously sends it. You can manage queue processing with the [STOP DISTRIBUTED SENDS](#query_language-system-stop-distributed-sends), [FLUSH DISTRIBUTED](#query_language-system-flush-distributed), and [START DISTRIBUTED SENDS](#query_language-system-start-distributed-sends) queries. You can also synchronously insert distributed data with the `insert_distributed_sync` setting.
 
 
 ### STOP DISTRIBUTED SENDS {#query_language-system-stop-distributed-sends}
 
-Disables background data distributing, when inserting data into the distributed tables.
+Disables background data distribution when inserting data into distributed tables.
```

Comment:
it's worth to mention what happens if they have already been stopped

Reply:
I have tried to send this query several times in a row, and there was no any effect, no exception nor something else. I don't know what to write.

Reply:
Has no effect if distributed sends have already been stopped?

Reply:
```
$ clickhouse-client 
ClickHouse client version 19.11.0.826.
Connecting to localhost:9000 as user default.
Connected to ClickHouse server version 19.11.0 revision 54423.

bayonet-VirtualBox :) system stop distributed sends

SYSTEM STOP DISTRIBUTED SENDS

Ok.

0 rows in set. Elapsed: 0.001 sec. 

bayonet-VirtualBox :) system stop distributed sends

SYSTEM STOP DISTRIBUTED SENDS

Ok.

0 rows in set. Elapsed: 0.001 sec. 
```